internal/provider/resources/resource_api_token.go

```diff
@@ -478,49 +477,6 @@ func (r *ApiTokenResource) ImportState(
 	resource.ImportStatePassthroughID(ctx, path.Root("id"), req, resp)
 }
 
-func (r *ApiTokenResource) ValidateConfig(
```

Comment:
so if we validate config we can't use `foreach` when creating a resource? will we need to change this for all resources then? 

also how can we validate the input and ensure the values passed in are correct?

Reply:
1. yes 
2. we cannot do it in this test and will have to rely on core APIs errors to do it